src/interface_r/R/auto_generated_wrappers.R

```diff
@@ -219,199 +215,119 @@ h2o4gpu.gradient_boosting_regressor <- function(
+	alphas = NULL,
+	lambdas = NULL,
+	double_precision = NULL,
+	order = NULL) {
 
-  model <- h2o4gpu$LogisticRegression(
-    penalty = penalty,
-    dual = dual,
-    tol = tol,
-    C = C,
-    fit_intercept = fit_intercept,
-    intercept_scaling = intercept_scaling,
-    class_weight = class_weight,
-    random_state = as_nullable_integer(random_state),
-    solver = solver,
-    max_iter = as.integer(max_iter),
-    multi_class = multi_class,
-    verbose = as.integer(verbose),
-    warm_start = warm_start,
-    n_jobs = as.integer(n_jobs),
+  model <- h2o4gpu$ElasticNetH2O(
+    n_threads = n_threads,
+    gpu_id = as.integer(gpu_id),
     n_gpus = as.integer(n_gpus),
-    glm_stop_early = glm_stop_early,
-    glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    backend = backend
-  )
-  h2o4gpu_model(model, c("classifier"))
-}
-
-#' @export
-h2o4gpu.lasso_regressor <- function(
-	alpha = 1.0,
-	fit_intercept = TRUE,
-	normalize = FALSE,
-	precompute = FALSE,
-	copy_X = TRUE,
-	max_iter = 5000L,
-	tol = 0.01,
-	warm_start = FALSE,
-	positive = FALSE,
-	random_state = NULL,
-	selection = "cyclic",
-	n_gpus = -1L,
-	glm_stop_early = TRUE,
-	glm_stop_early_error_fraction = 1.0,
-	verbose = FALSE,
-	backend = "auto") {
-
-  model <- h2o4gpu$Lasso(
-    alpha = alpha,
     fit_intercept = fit_intercept,
-    normalize = normalize,
-    precompute = precompute,
-    copy_X = copy_X,
-    max_iter = as.integer(max_iter),
+    lambda_min_ratio = lambda_min_ratio,
+    n_lambdas = as.integer(n_lambdas),
+    n_folds = as.integer(n_folds),
+    n_alphas = as.integer(n_alphas),
     tol = tol,
-    warm_start = warm_start,
-    positive = positive,
-    random_state = as_nullable_integer(random_state),
-    selection = selection,
-    n_gpus = as.integer(n_gpus),
+    tol_seek_factor = tol_seek_factor,
+    lambda_stop_early = lambda_stop_early,
     glm_stop_early = glm_stop_early,
     glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    verbose = verbose,
-    backend = backend
-  )
-  h2o4gpu_model(model, c("regressor"))
-}
-
-#' @export
-h2o4gpu.ridge_regressor <- function(
-	alpha = 1.0,
-	fit_intercept = TRUE,
-	normalize = FALSE,
-	copy_X = TRUE,
-	max_iter = 5000L,
-	tol = 0.01,
-	solver = "auto",
-	random_state = NULL,
-	n_gpus = -1L,
-	glm_stop_early = TRUE,
-	glm_stop_early_error_fraction = 1.0,
-	verbose = FALSE,
-	backend = "auto") {
-
-  model <- h2o4gpu$Ridge(
-    alpha = alpha,
-    fit_intercept = fit_intercept,
-    normalize = normalize,
-    copy_X = copy_X,
     max_iter = as.integer(max_iter),
-    tol = tol,
-    solver = solver,
-    random_state = as_nullable_integer(random_state),
-    n_gpus = as.integer(n_gpus),
-    glm_stop_early = glm_stop_early,
-    glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    verbose = verbose,
-    backend = backend
+    verbose = as.integer(verbose),
+    family = "elasticnet",
+    store_full_path = as.integer(store_full_path),
```

Comment:
Same comment as elsewhere, this should be changed to boolean in Python (then here).